src/Command/VersionCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\lightning\Command;
+
+use Drupal\Console\Core\Command\Command;
+use Drupal\Console\Core\Style\DrupalStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Yaml\Yaml;
+use Drupal\Console\Annotations\DrupalCommand;
+
+/**
+ * Class VersionCommand.
+ *
+ * @DrupalCommand
+ */
+class VersionCommand extends Command {
+
+  /**
+   * The Drupal application root.
+   *
+   * @var string
+   */
+  protected $appRoot;
+
+  /**
+   * VersionCommand constructor.
+   *
+   * @param string $app_root
+   *   The Drupal application root.
+   */
+  public function __construct($app_root) {
+    parent::__construct('lightning:version');
+    $this->appRoot = $app_root;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this->setName('lightning:version');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $finder = (new Finder())
+      ->files()
+      ->name('lightning.info.yml')
+      ->in($this->appRoot . '/profiles');
+
+    foreach ($finder as $info_file) {
+      $info = Yaml::parse($info_file->getContents());
+      if (($info['distribution']['name'] == 'Lightning') && isset($info['version'])) {
+        // There should only be one file in docroot/profiles named
+        // lightning.info.yml, but just to make sure we have the right file in
+        // the iterator, break out when we have enough information to be
+        // reasonably confident.
+        break;
+      }
+    }
+
+    $io = new DrupalStyle($input, $output);
+    $io->info($this->toSemanticVersion($info['version']));
```

Comment:
There's no guarantee that $info actually exists. We should probably add an if (isset($info)) check here.